leader-election-impl/src/main/java/com/palantir/paxos/SingleLeaderPinger.java

```diff
@@ -72,15 +73,17 @@ public LeaderPingResult pingLeaderWithUuid(UUID uuid) {
         MultiplexingCompletionService<LeaderPingerContext<PingableLeader>, Boolean> multiplexingCompletionService
                 = MultiplexingCompletionService.create(leaderPingExecutors);
 
-        multiplexingCompletionService.submit(leader, () -> leader.pinger().ping());
-
         try {
+            multiplexingCompletionService.submit(leader, () -> leader.pinger().ping());
             Future<Map.Entry<LeaderPingerContext<PingableLeader>, Boolean>> pingFuture = multiplexingCompletionService
                     .poll(leaderPingResponseWait.toMillis(), TimeUnit.MILLISECONDS);
             return getLeaderPingResult(uuid, pingFuture);
         } catch (InterruptedException ex) {
             Thread.currentThread().interrupt();
             return LeaderPingResults.pingCallFailure(ex);
+        } catch (RejectedExecutionException ex) {
```

Comment:
nit: while this follows the above, normal usage is `e` instead of `ex`, including in other places in this PR.

Reply:
Yeah, I just copied the above. I'll change both.